allure-ruby-commons/spec/unit/allure_spec.rb

```diff
@@ -164,5 +164,12 @@ def lifecycle
         expect(last_step.status_details.message).to eq("Error")
       end
     end
+
+    it "adds parameter" do
+      result = allure.run_step("New step") do
```

Comment:
```suggestion
      allure.run_step("New step") do
```

Since `result` is not used, it fails rubocop check at the moment.

Reply:
Removed it in https://github.com/allure-framework/allure-ruby/pull/347/commits/35cf43fa06e4f647802ccd4afea35bd7ef692acb :)